src/utils/getArticleById.js

```diff
@@ -0,0 +1,14 @@
+import axios from "axios"
+
+function getArticleById(article_id) {
+        return axios
+        .get(`https://robbies-articles-website.onrender.com/api/articles/${article_id}`)
+        .then((response) => {
+            return response.data.article[0]
+        })
+        .catch((err) => {
```

Comment:
Catch blocks should be used at the end of a promise chain, which will be in the file where getByArticleId is initially invoked